### PR TITLE
linux: add windowing system to SystemInfo screen

### PR DESCRIPTION
**Pull request by wsnipex**
## Description

This adds the used windowing system to the system info screen

alt text

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [X] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [ ] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -22355,3 +22355,8 @@ msgstr ""
 msgctxt "#39152"
 msgid "Integer scaling (IS) is a Nearest-Neighbor(NN) upscaling technique using Display Engine"
 msgstr ""
+
+#: xbmc/windows/GUIWindowSystemInfo.cpp
+msgctxt "#39153"
+msgid "Window Manager:"
```
Comment (eigendude) [inline addons/resource.language.en_gb/resources/strings.po:22361]: `Window manager:` for case-consistency (the other labels use APA Style sentence case)

Comment (garbear) [inline addons/resource.language.en_gb/resources/strings.po:22361]: `Window manager:` for case-consistency (the other labels use APA Style sentence case)

Comment (wsnipex) [reply addons/resource.language.en_gb/resources/strings.po]: Label changed to "Windowing system:" as suggested on slack

**Comment by wsnipex** [comment]
freebsd error unrelated, jenkins build this please

**Pull request merged**